<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestQueryConsole</class>
 <widget class="QWidget" name="TestQueryConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>17</height>
      </size>
     </property>
     <property name="text">
      <string>API Call:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLineEdit" name="apiCallInput">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>27</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>POST Args:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="postVarKey"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QPlainTextEdit" name="encodedPostArgs"/>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>400</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>489</width>
        <height>398</height>
       </rect>
      </property>
      <widget class="QTextBrowser" name="outputFrame">
       <property name="geometry">
        <rect>
         <x>-5</x>
         <y>-9</y>
         <width>501</width>
         <height>411</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QPushButton" name="sendButton">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="postVarValue"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>apiCallInput</tabstop>
  <tabstop>postVarKey</tabstop>
  <tabstop>postVarValue</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>encodedPostArgs</tabstop>
  <tabstop>sendButton</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>outputFrame</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
